ChatApp/src/screens/Profile/Style.tsx

```diff
@@ -0,0 +1,152 @@
+import { StyleSheet } from 'react-native';
+
+const Styles = StyleSheet.create({
+  Header_Container: {
+    height: 30,
+    width: '100%',
+    marginTop: 10,
+    // backgroundColor: 'red',
+    flexDirection: 'row',
+  },
+  Icon_Container: {
+    // backgroundColor: 'green',
+    width: '10%',
+    marginLeft: 20,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  Heading: {
+    // backgroundColor: 'yellow',
+    width: '60%',
+    alignItems: 'center',
+  },
+  Heading_Text: {
+    fontWeight: '500',
+    fontSize: 20,
+  },
+  Content: {
+    height: '78%',
+    marginTop: 50,
+    // backgroundColor: 'yellow',
+  },
+  Image_Container: {
+    height: 200,
+    justifyContent: 'center',
+    alignItems: 'center',
+    // backgroundColor: 'green',
+  },
+  Image_Box: {
+    width: 200,
+    height: 200,
+    justifyContent: 'center',
+    alignItems: 'center',
+    // backgroundColor: 'red',
+  },
+  Input_Container: {
+    height: 350,
+    justifyContent: 'space-evenly',
+    alignItems: 'center',
+  },
+  Button_Box: {
+    width: '100%',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  Button: {
+    width: '90%',
+  },
+  images: {
+    height: '100%',
+    width: '100%',
+    borderRadius: 500,
+  },
+  Edit_Icon: {
+    backgroundColor: '#fff',
+    borderRadius: 300,
+    marginTop: -60,
```

Comment:
Why is this needed?

Reply:
It is the button, which is used for selecting Image.